updating/updating-clusters-sro.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="updating-clusters-sro"]
+= Updating a cluster that includes the SRO
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-sro
+
+toc::[]
+
+When updating a cluster that includes the SRO, it is important to consider whether the new kernel module version will be compatible with the kernel modules currently loaded by the SRO. You can run a preflight check to confirm that the SRO will be able to upgrade the kernel modules.
+
+include::modules/update-upgrading-sro.adoc[leveloffset=+1]
```

Comment:
After this line, you could add an "Additional resources" section that provides a cross-reference to https://docs.openshift.com/container-platform/4.11/hardware_enablement/psap-special-resource-operator.html. This will provide a useful reference for the reader to general information about the Special Resource Operator.